lib/util.js

```diff
@@ -1,37 +1,41 @@
 // Copyright (c) 2019 Magic Leap, Inc. All Rights Reserved
 // Distributed under Apache 2.0 License. See LICENSE file in the project root for full license information.
-let fs = require('fs');
-let glob = require('glob');
-let { exec, execFile, execSync, spawn } = require('child_process');
+let fs = require("fs");
+let glob = require("glob");
+let { exec, execFile, execSync, spawnSync, spawn } = require("child_process");
+const path = require("path");
+const rename = require.resolve('react-native-rename');
 
-module.exports.findPackageName = function () {
-  let manifestPath = 'manifest.xml';
-  var packageName = '';
+module.exports.findPackageName = function() {
+  let manifestPath = "manifest.xml";
+  var packageName = "";
```

Comment:
why change the quotes?

Reply:
This is related to semistandard library for lint. I have Prettier installed in VsCode and when I'm formatting with VSCode it is changing single quotes to double quotes. I just changed the rule in Prettier and I formatted command files back to single quotes